src/index.js

```diff
@@ -0,0 +1,204 @@
+        <g id="Group" fill="none" fill-rule="evenodd" transform="translate(467 392)">
+          <path id="heart" d="M29.144 20.773c-.063-.13-4.227-8.67-11.44-2.59C7.63 28.795 28.94 43.256 29.143 43.394c.204-.138 21.513-14.6 11.44-25.213-7.214-6.08-11.377 2.46-11.44 2.59z" fill="#AAB8C2"/>
+          <circle id="main-circ" fill="#E2264D" opacity="0" cx="29.5" cy="29.5" r="1.5"/>
+
+          <g id="grp7" opacity="0" transform="translate(7 6)">
+            <circle id="oval1" fill="#9CD8C3" cx="2" cy="6" r="2"/>
+            <circle id="oval2" fill="#8CE8C3" cx="5" cy="2" r="2"/>
+          </g>
+
+
+          <g id="grp6" opacity="0" transform="translate(0 28)">
+            <circle id="oval1" fill="#CC8EF5" cx="2" cy="7" r="2"/>
+            <circle id="oval2" fill="#91D2FA" cx="3" cy="2" r="2"/>
+          </g>
+
+          <g id="grp3" opacity="0" transform="translate(52 28)">
+            <circle id="oval2" fill="#9CD8C3" cx="2" cy="7" r="2"/>
+            <circle id="oval1" fill="#8CE8C3" cx="4" cy="2" r="2"/>
+          </g>
+
+          <g id="grp2" opacity="0" transform="translate(44 6)">
+            <circle id="oval2" fill="#CC8EF5" cx="5" cy="6" r="2"/>
+            <circle id="oval1" fill="#CC8EF5" cx="2" cy="2" r="2"/>
+          </g>
+
+          <g id="grp5" opacity="0" transform="translate(14 50)">
+            <circle id="oval1" fill="#91D2FA" cx="6" cy="5" r="2"/>
+            <circle id="oval2" fill="#91D2FA" cx="2" cy="2" r="2"/>
+          </g>
+
+          <g id="grp4" opacity="0" transform="translate(35 50)">
+            <circle id="oval1" fill="#F48EA7" cx="6" cy="5" r="2"/>
+            <circle id="oval2" fill="#F48EA7" cx="2" cy="2" r="2"/>
+          </g>
+
+          <g id="grp1" opacity="0" transform="translate(24)">
+            <circle id="oval1" fill="#9FC7FA" cx="2.5" cy="3" r="2"/>
+            <circle id="oval2" fill="#9FC7FA" cx="7.5" cy="2" r="2"/>
+          </g>
+        </g>
+      </svg>`;
+
+const displayCommentsList = async (data) => {
+  const commentsList = document.querySelector('.comments-list');
+  commentsList.innerHTML = '';
+  if (data.length) {
+    data.forEach((comment) => {
+      commentsList.innerHTML += `<li class='comment'>
+        <p>${comment.creation_date}<p/>
+        <p>${comment.username}<p/> 
+        <p>${comment.comment}<p/>
+      <li/>`;
+    });
+  } else commentsList.innerHTML = '<li>No comments</li>';
+};
+
+const displayPopup = async (item, id) => {
+  const data = await getCommentsList(id);
+  const modal = document.querySelector('.overlay');
+  const img = document.querySelector('.modal-img img');
+  const productTitle = document.querySelector('.product-title');
+  form.parentElement.setAttribute('data-set', id);
+  img.src = item ? item.strMealThumb : placeholderImg;
+  productTitle.textContent = item.strMeal;
+  modal.classList.add('active-modal');
+  commentsCounter(data);
+  displayCommentsList(data);
+};
```

Comment:
Kindly make sure that your project sticks as much as possible to the template provided. 

- I think it is a good idea to add some details about the food in your pop-up windows. :pray: :pray: 

Your design
![image](https://user-images.githubusercontent.com/98363075/204060395-9af4a88a-5409-4ac2-9e41-5b4b7f6543c3.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/204060477-57f16905-8b3e-41dd-8681-2e5e490b9598.png)

- Kindly make sure that your pop-up windows are being rendered completely. It helps to improve the user's experience.
![image](https://user-images.githubusercontent.com/98363075/204060807-11912a36-ae08-4125-a6a1-b20c4e6c2742.png)
![image](https://user-images.githubusercontent.com/98363075/204060846-cec5ea77-f839-40ca-b82d-188c8d602588.png)